src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -0,0 +1,128 @@
+package com.minecolonies.achievements;
+
+import com.minecolonies.MineColonies;
+import com.minecolonies.blocks.ModBlocks;
+import com.minecolonies.items.ModItems;
+import com.minecolonies.lib.Constants;
+import net.minecraft.init.Items;
+import net.minecraft.stats.Achievement;
+import net.minecraftforge.common.AchievementPage;
+
+/**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)